### PR TITLE
deepfry

### PR DESCRIPTION
**Pull request by ksdfg**
Gave up on flares for now since design seems to have changed.

### CODE REVIEW

**Review by akhilnarang — DISMISSED** [review]

**Review by akhilnarang — DISMISSED** [review]

File: telebot/modules/memes.py [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+from io import BytesIO
+from os import remove
+from os.path import join
+from pathlib import Path
 from random import choice, randint
 from re import sub
+from uuid import uuid4
 
+from PIL import Image
+from cv2.cv2 import CascadeClassifier, cvtColor, COLOR_BGR2GRAY
```
Comment (akhilnarang) [inline telebot/modules/memes.py:11]: Unused imports?

**Comment by ksdfg** [comment]
Eeeeek, leftover stuff from experiments

**Comment by akhilnarang** [comment]
Fixing

**Review by akhilnarang — APPROVED** [review]

**Comment by ksdfg** [comment]
Reverted, cv2 error on heroku

**Pull request merged**